doc/getting_started.md

```diff
@@ -537,64 +558,23 @@ utilization (FPGA build) will all affect the safe maximum gain setting.
 To perform a SHA3 (KMAC) capture, use this command:
 ```console
 $ cd cw/cw305
-$ ./simple_capture_traces.py --cfg-file capture_sha3.yaml capture sha3-random --num-traces 100 --plot-traces 10
+$ ./capture.py --cfg-file capture_sha3.yaml capture sha3-random --num-traces 100 --plot-traces 10
 ```
 
 The above command will send SHA3 (KMAC) requests with a fixed key and random 
 texts. In order to capture traces for DTR TVLA Section 5.3: "General Test: 
 Fixed-vs.-Random Key Datasets", following command may be used:
 ```console
 $ cd cw/cw305
-$ ./simple_capture_traces.py --cfg-file capture_sha3.yaml capture sha3-fvsr-key --num-traces 100 --plot-traces 10
+$ ./capture.py --cfg-file capture_sha3.yaml capture sha3-fvsr-key --num-traces 100 --plot-traces 10
 ```
 
-
 You should see similar output as in the AES example. Once the power traces have
 been collected, a picture similar to the following should be shown in your
 browser.
 
 ![](img/sample_traces_sha3.png)
 
-### AES Batch Capture
-
-To capture AES power traces in batch mode,
-
-```console
-$ cd cw/cw305
-$ ./simple_capture_traces_batch.py --scope cw
-```
-
-This should produce console output like
-```console
-Connecting and loading FPGA... Done!
-Initializing PLL1
-Programming OpenTitan with "objs/aes_serial_fpga_cw310.bin"...
-Transferring frame 0x00000000 @             0x00000000.
-Transferring frame 0x00000001 @             0x000007D8.
-Transferring frame 0x00000002 @             0x00000FB0.
-Transferring frame 0x00000003 @             0x00001788.
-Transferring frame 0x00000004 @             0x00001F60.
-Transferring frame 0x00000005 @             0x00002738.
-Transferring frame 0x80000006 @             0x00002F10.
-Scope setup with sampling rate 100004653.0 S/s
-Reading from FPGA using simpleserial protocol.
-Target simpleserial version: z01 (attempts: 2).
-Connected to ChipWhisperer (num_samples: 740, num_samples_actual: 740, num_segments_actual: 1)
-Using key: '2b7e151628aed2a6abf7158809cf4f3c'.
-Capturing: 100%|█████████████████████| 5000/5000 [00:03<00:00, 1301.86 traces/s]
-```
-
-By default, this will capture 5000 traces which should be sufficient to make
-sure the setup is configured and working as expected before starting a long
-running capture. The plot should look very similar to the one of the
-non-batched AES capture above.
-
-To capture e.g. 2 Mio traces, open the config file `capture_aes.yaml` and change
-`capture.num_traces` to `2000000`. Also, it is recommended to set
-`plot_capture.show` to `false` as the routine checking the ADC gain setting
-(as part of the plot generation) can take a long time for high number of traces.
-
-
 ## Performing Example SCA Attack on AES with Masking Disabled
```

Comment:
In this section can you please also remove steps 3 (set capture.num_traces) and 4 (set plot_capture.show) and instead adjust the capture command below?

Reply:
Thanks.